addons/mail/static/src/js/chatter.js

```diff
@@ -435,7 +435,11 @@ var Chatter = Widget.extend({
      */
      _updateAttachmentCounter: function () {
         var count = this.record.data.message_attachment_count || 0;
-        this.$('.o_chatter_attachment_button_count').html(count);
+        var element = this.$('.o_chatter_attachment_button_count');
+        if (+element.html() != count) {
```

Comment:
- remove `+`
- `!=` => `!==`

Reply:
html() will return a string, so it is either +, either !=

Reply:
Or `Number(element.html()) !== count`

Reply:
+ is shorter :ppp

What's the difference between `+` and `Number` ??

Reply:
`+<string>` is equivalent to `Number(<string>)`, but the latter is preferred because not many devs know what the former means and it can be hard to read.